Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+
+public class AudioController : MonoBehaviour {
+
+    private float maxVolume;
+	private float fadeTime;
+
+	public bool hasPlayer;
+
+    private BoxCollider2D audioRangeCollider;
+    private AudioSource audioSource;
+    public List<GameObject> dolls = new List<GameObject>();
+
+    // Use this for initialization
+    void Start() {
+        audioRangeCollider = GetComponent<BoxCollider2D>();
+        audioSource = GetComponent<AudioSource>();
+		maxVolume = MechanicAudioManager.getInstance ().maxVolume;
```

Comment:
We should not make the max volume the same for every type of audio. We should seperate these by the type of audio (because the volume is different for each sound clip).